src/DynamoCore/Graph/Nodes/NodeInputData.cs

```diff
@@ -21,16 +32,21 @@ public enum NodeInputTypes
         colorInput,
         [EnumMember(Value = "date")]
         dateInput,
-        [Obsolete("Use hostSelection or dropdownSelection instead")]
+        [Obsolete()]
         [EnumMember(Value = "selection")]
         selectionInput,
+        /// <summary>
+        /// Should not be used when setting NodeInputData.Type, use selection instead, can be used with Type2.
+        /// </summary>
         [EnumMember(Value = "hostSelection")]
         hostSelection,
+        /// <summary>
+        /// Should not be used when setting NodeInputData.Type, use selection instead, can be used with Type2.
+        /// </summary>
```

Comment:
It just occurred to me that maybe we could have used the `[NonSerialized()]` attribute for the  `hostSelection` and `dropdownSelection` members. This would only make sense with 2 (duplicate) enum classes